redash/models.py

```diff
@@ -947,6 +949,7 @@ def fork(self, user):
         kwargs = {a: getattr(self, a) for a in forked_list}
         forked_query = Query.create(name=u'Copy of (#{}) {}'.format(self.id, self.name),
                                     user=user, **kwargs)
+        forked_query.record_changes(changed_by=user)
```

Comment:
This is the only place you refer to the concept of forking. Curious: what ends up getting written to the `changes` database table in this case?